TIL/daily/2025-03-21.md

```
@@ -0,0 +1,245 @@
+- h2는 heroes에 추가되지 않았기에 .remove()해도 변경되는 것은 없다
+- Set도 마찬가지
+
+### == 연산자 재정의
+
+- 재정의하여 동등성 규칙을 재정의할 수 있다
+- List에서 동등성 비교시 사용됨
+
+```dart
+@override
+bool operator ==(Object other) =>
+	identical(this, other) ||
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)